config/config.go

```diff
@@ -12,6 +14,9 @@ type Config struct {
 	BindAddr           string        `envconfig:"BIND_ADDR"`
 	CORSAllowedOrigins string        `envconfig:"CORS_ALLOWED_ORIGINS"`
 	ShutdownTimeout    time.Duration `envconfig:"SHUTDOWN_TIMEOUT"`
+	SVG2PNGExecutable  string        `envconfig:"SVG_2_PNG_EXECUTABLE"`
+	SVG2PNGArgLine     string        `envconfig:"SVG_2_PNG_ARG_LINE"`
+	SVG2PNGArguments   []string
```

Comment:
Should this have an envconfig tag?

Reply:
SVG2PNGArguments is parsed from SVG2PNGArgLine - unless you know a better way to get an array from environment variables?